### ISSUE TITLE
Navigator no longer shows installed package versions or whether package version is latest

### ISSUE THREAD

**Issue by DManowitz** [ContinuumIO/anaconda-issues#13028]
### Checklist

- [X] I added a descriptive title
- [X] I searched for other issues and couldn't find a solution or duplication
- [X] I already searched in Google and didn't find any good information or help
- [X] I looked at the [docs](https://docs.anaconda.com/anaconda/navigator/) and didn't see anything to help

### What happened?

On Windows 10, I previously had Navigator working both with an Anaconda install, and later with Miniconda and Navigator added into my base environment with a similar setup to what I have now.  However, I was having some issues, so I uninstalled Miniconda and reinstalled it and added Navigator back into my base environment.

Now, in the Environments view, I no longer see the column that shows the version of each installed package (and whether or not the installed package is the latest version).  I also see "100 " before all of my package names.  I'm guessing this is related, though I'm not 100% sure.

If I right-click on the checkbox to install or modify a package, I still see all available versions of the package, though.

### Conda info

```shell
active environment : base
    active env location : C:\Users\manow\miniconda3
            shell level : 1
       user config file : C:\Users\manow\.condarc
 populated config files : C:\Users\manow\.condarc
          conda version : 4.14.0
    conda-build version : not installed
         python version : 3.9.12.final.0
       virtual packages : __win=0=0
                          __archspec=1=x86_64
       base environment : C:\Users\manow\miniconda3  (writable)
      conda av data dir : C:\Users\manow\miniconda3\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/pytorch/win-64
                          https://conda.anaconda.org/pytorch/noarch
                          https://conda.anaconda.org/pyg/win-64
                          https://conda.anaconda.org/pyg/noarch
                          https://conda.anaconda.org/nvidia/label/cuda-11.3.1/win-64
                          https://conda.anaconda.org/nvidia/label/cuda-11.3.1/noarch
                          https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/intel/win-64
                          https://conda.anaconda.org/intel/noarch
                          https://repo.anaconda.com/pkgs/main/win-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/win-64
                          https://repo.anaconda.com/pkgs/r/noarch
                          https://repo.anaconda.com/pkgs/msys2/win-64
                          https://repo.anaconda.com/pkgs/msys2/noarch
          package cache : C:\Users\manow\miniconda3\pkgs
                          C:\Users\manow\.conda\pkgs
                          C:\Users\manow\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\manow\miniconda3\envs
                          C:\Users\manow\.conda\envs
                          C:\Users\manow\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/4.14.0 requests/2.28.1 CPython/3.9.12 Windows/10 Windows/10.0.19044
          administrator : False
             netrc file : None
           offline mode : False
```

### Conda config

```shell
==> C:\Users\manow\.condarc <==
channel_priority: disabled
channels:
  - pytorch
  - pyg
  - nvidia/label/cuda-11.3.1
  - conda-forge
  - intel
  - defaults
```

### Conda list

```shell
# packages in environment at C:\Users\manow\miniconda3:
#
# Name                    Version                   Build  Channel
anaconda-client           1.10.0           py39haa95532_0    defaults
anaconda-navigator        2.2.0            py39haa95532_0    defaults
attrs                     21.4.0             pyhd3eb1b0_0    defaults
brotlipy                  0.7.0           py39h2bbff1b_1003    defaults
bzip2                     1.0.8            vc14h53ad9d4_9  [vc14]  intel
ca-certificates           2022.07.19           haa95532_0    defaults
certifi                   2022.6.15        py39hcbf5309_0    conda-forge
cffi                      1.15.1           py39h2bbff1b_0    defaults
chardet                   4.0.0           py39haa95532_1003    defaults
charset-normalizer        2.0.4              pyhd3eb1b0_0    defaults
clyent                    1.2.2            py39haa95532_1    defaults
colorama                  0.4.5            py39haa95532_0    defaults
conda                     4.14.0           py39haa95532_0    defaults
conda-content-trust       0.1.3            py39haa95532_0    defaults
conda-package-handling    1.8.1            py39h8cc25b3_0    defaults
conda-repo-cli            1.0.5            py39haa95532_0    defaults
conda-token               0.3.0              pyhd3eb1b0_0    defaults
conda-tree                1.0.5              pyhd8ed1ab_0    conda-forge
console_shortcut          0.1.1                         4    defaults
cryptography              37.0.1           py39h21b164f_0    defaults
cytoolz                   0.11.0           py39h2bbff1b_0    defaults
freetype                  2.10.4               hd328e21_0    defaults
glib                      2.69.1               h5dc1a3c_1    defaults
gst-plugins-base          1.18.5               h9e645db_0    defaults
gstreamer                 1.18.5               hd78058f_0    defaults
icu                       58.2                 ha925a31_3    defaults
idna                      3.3                pyhd3eb1b0_0    defaults
intelpython               2022.1.0                      0    intel
jpeg                      9e                   h2bbff1b_0    defaults
jsonschema                4.4.0            py39haa95532_0    defaults
jupyter_core              4.10.0           py39haa95532_0    defaults
lerc                      3.0                  hd77b12b_0    defaults
libarchive                3.5.2                h214662b_0    defaults
libclang                  12.0.0          default_h627e005_2    defaults
libcurl                   7.84.0               h86230a5_0    defaults
libdeflate                1.8                  h2bbff1b_5    defaults
libffi                    3.4.2                hd77b12b_4    defaults
libiconv                  1.16                 h2bbff1b_2    defaults
libmamba                  0.25.0               h81a967f_2    conda-forge
libmambapy                0.25.0           py39h4126fcf_2    conda-forge
libogg                    1.3.5                h2bbff1b_1    defaults
libpng                    1.6.37               h2a8f88b_0    defaults
libsolv                   0.7.22               h7755175_0    conda-forge
libssh2                   1.10.0               hcd4344a_0    defaults
libtiff                   4.4.0                h8a3f274_0    defaults
libvorbis                 1.3.7                he774522_0    defaults
libwebp                   1.2.2                h2bbff1b_0    defaults
libxml2                   2.9.14               h0ad7f3c_0    defaults
libxslt                   1.1.35               h2bbff1b_0    defaults
libzlib                   1.2.12               h8ffe710_2    conda-forge
lz4-c                     1.9.3                h2bbff1b_1    defaults
mamba                     0.25.0           py39hb3d9227_2    conda-forge
menuinst                  1.4.18           py39h59b6b97_0    defaults
navigator-updater         0.2.1                    py39_1    defaults
nbformat                  5.3.0            py39haa95532_0    defaults
networkx                  2.8.6              pyhd8ed1ab_0    conda-forge
openssl                   1.1.1q               h8ffe710_0    conda-forge
packaging                 21.3               pyhd3eb1b0_0    defaults
pathlib                   1.0.1              pyhd3eb1b0_1    defaults
pcre                      8.45                 hd77b12b_0    defaults
pillow                    9.2.0            py39hdc2b20a_1    defaults
pip                       22.1.2           py39haa95532_0    defaults
pipdeptree                2.2.0              pyhd8ed1ab_0    conda-forge
ply                       3.11             py39haa95532_0    defaults
powershell_shortcut       0.0.1                         3    defaults
psutil                    5.9.0            py39h2bbff1b_0    defaults
pybind11-abi              4                    hd8ed1ab_3    conda-forge
pycosat                   0.6.3            py39h2bbff1b_0    defaults
pycparser                 2.21               pyhd3eb1b0_0    defaults
pyjwt                     2.4.0            py39haa95532_0    defaults
pyopenssl                 22.0.0             pyhd3eb1b0_0    defaults
pyparsing                 3.0.4              pyhd3eb1b0_0    defaults
pyqt                      5.15.7           py39hd77b12b_0    defaults
pyqt5-sip                 12.11.0          py39hd77b12b_0    defaults
pyrsistent                0.18.0           py39h196d8e1_0    defaults
pysocks                   1.7.1            py39haa95532_0    defaults
python                    3.9.12               h6244533_0    defaults
python-dateutil           2.8.2              pyhd3eb1b0_0    defaults
python-fastjsonschema     2.15.1             pyhd3eb1b0_0    defaults
python_abi                3.9                      2_cp39    conda-forge
pytz                      2022.1           py39haa95532_0    defaults
pywin32                   302              py39h2bbff1b_2    defaults
pyyaml                    6.0              py39h2bbff1b_1    defaults
qt-main                   5.15.2               he8e5bd7_7    defaults
qt-webengine              5.15.9               hb9a9bb5_4    defaults
qtpy                      2.0.1              pyhd3eb1b0_0    defaults
qtwebkit                  5.212                h3ad3cdb_4    defaults
reproc                    14.2.4               hd77b12b_1    defaults
reproc-cpp                14.2.4               hd77b12b_1    defaults
requests                  2.28.1           py39haa95532_0    defaults
ruamel_yaml               0.15.100         py39h2bbff1b_0    defaults
setuptools                63.4.1           py39haa95532_0    defaults
sip                       6.6.2            py39hd77b12b_0    defaults
six                       1.16.0             pyhd3eb1b0_1    defaults
sqlite                    3.39.2               h2bbff1b_0    defaults
tk                        8.6.12               h2bbff1b_0    defaults
toml                      0.10.2             pyhd3eb1b0_0    defaults
toolz                     0.11.2             pyhd3eb1b0_0    defaults
tqdm                      4.64.0           py39haa95532_0    defaults
traitlets                 5.1.1              pyhd3eb1b0_0    defaults
tzdata                    2022a                hda174b7_0    defaults
ujson                     5.4.0            py39hd77b12b_0    defaults
urllib3                   1.26.11          py39haa95532_0    defaults
vc                        14.2                 h21ff451_1    defaults
vs2015_runtime            14.27.29016          h5e58377_2    defaults
wheel                     0.37.1             pyhd3eb1b0_0    defaults
win_inet_pton             1.1.0            py39haa95532_0    defaults
wincertstore              0.2              py39haa95532_2    defaults
xz                        5.2.5                h8cc25b3_1    defaults
yaml                      0.2.5                he774522_0    defaults
yaml-cpp                  0.7.0                hd77b12b_1    defaults
zlib                      1.2.12               h8cc25b3_2    defaults
zstd                      1.5.2                h19a0ad4_0    defaults
```

**Comment by vshevchenko-anaconda** [comment]
A possible fix for this issue is included in Navigator 2.3.0. We'll notify you when a new version of Navigator is available.

**Comment by vshevchenko-anaconda** [comment]
Navigator 2.3.0 is released. Please reopen if you are still facing this issue.
